### ISSUE TITLE
Upgrading from v0.x to v1.0 docs

### ISSUE THREAD

**Issue by dawzrd** [tailwindcss/tailwindcss#918]
You may want to mention in your 1.0 upgrade guide that all color names have changed. In the examples throughout the docs, colors are using the non-numeric-suffixed-old-legacy-name.

Its not until [https://tailwindcss.com/docs/customizing-colors/#naming-your-colors](https://tailwindcss.com/docs/customizing-colors/#naming-your-colors)

that we finally find out why all of our colors are missing.

**Comment by MichaelDeBoey** [comment]
I think you looked over https://tailwindcss.com/docs/upgrading-to-v1#2-9-inline-your-colors-variable-into-theme-colors

**Comment by adamwathan** [comment]
Yeah the upgrade guide assumes you have a config file already which has all your colors defined (all pre-1.0 projects were that way), and the upgrade guide explains how to upgrade your config file and keep all those colors.

The upgrade guide also has a section on updating all your color names if you didn't have your own config file:

https://tailwindcss.com/docs/upgrading-to-v1/#1-update-any-usage-of-text-bg-border-color-classes

Let me know what I could do to make it clearer, but everything should work perfectly if you follow every step one by one and don't skip over anything.
